### PR TITLE
Update vep.cwl to use v90

### PR DESCRIPTION
**Pull request by susannasiebert**
This also brings the master version of vep.cwl in line with the toil_compatibility branch.

### CODE REVIEW

File: detect_variants/vep.cwl [hunk 1]
```diff
@@ -3,22 +3,20 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Ensembl Variant Effect Predictor"
-baseCommand: ["/usr/bin/perl", "/usr/bin/variant_effect_predictor.pl"]
-
+baseCommand: ["/home/vep/src/ensembl-vep/vep", "-I", "/home/vep/Plugins"]
 requirements:
+    - class: ShellCommandRequirement
```
Comment (tmooney) [inline detect_variants/vep.cwl]: (I see it's in the `toil_compatibility` version, too, but) what's using the `ShellCommandRequirement`?

Comment (susannasiebert) [reply detect_variants/vep.cwl]: I'm always confused when to use which requirements. I think I might've added it for the `cache_dir` and `hgvs` `valueFrom` inputs. But I guess it's not needed for that?

Comment (tmooney) [reply detect_variants/vep.cwl]: Yeah, the `valueFrom` expressions are the `InlineJavascriptRequirement`.  The most common reason we need `ShellCommandRequirement` is for pipes.

Comment (susannasiebert) [reply detect_variants/vep.cwl]: Ok, I removed it.

**Pull request merged**